tests/annotations/test_annotations.py

```diff
@@ -0,0 +1,642 @@
+    assert_frame_equal(written_results, expected_results, check_exact = False)
+
+
+@pytest.mark.parametrize(
+    "test_data_name_dir, gtf_file, annotations, gene_id_file, expected",
+    [
+        (   "filter_by_exon_distance_small",
+            "gencode.v44.annotation.gtf.gz",
+            "annotations.parquet",
+            "protein_coding_genes.parquet",
+            "expected.parquet",
+        ),
+    ]
+)
+def test_filter_by_exon_distance(
+     test_data_name_dir, gtf_file, annotations, gene_id_file, expected, tmp_path
+):
+    current_test_data_dir = tests_data_dir / 'filter_by_exon_distance' / test_data_name_dir
+    gtf_file_path = current_test_data_dir / 'input' /  gtf_file
+    annotations_path = current_test_data_dir / 'input' / annotations
+    gene_id_path = current_test_data_dir / 'input' / gene_id_file
+
+    expected_path = current_test_data_dir / 'expected' / expected
+    output_path = tmp_path / 'out.parquet'
+    cli_runner = CliRunner()
+    cli_parameters = [
+        'filter-annotations-by-exon-distance',
+        annotations_path.as_posix(),
+        gtf_file_path.as_posix(),
+        gene_id_path.as_posix(),
+        output_path.as_posix(),
+        ]
+    result = cli_runner.invoke(annotations_cli, cli_parameters, catch_exceptions=False)
+    assert result.exit_code == 0
+    written_results = pd.read_parquet(output_path)
+    expected_results = pd.read_parquet(expected_path)
+    assert written_results.shape == expected_results.shape
+    assert_frame_equal(written_results, expected_results, check_exact = False)
+
+
+@pytest.mark.parametrize(
+    "test_data_name_dir, yaml_file, annotations, expected",
+    [
+        (   "select_rename_fill_columns_small",
+            "annotation_colnames_filling_values.yaml",
+            "annotations.parquet",
+            "expected.parquet",
+        ),
+    ]
+)
+def test_select_rename_fill_annotations(
+     test_data_name_dir, yaml_file, annotations, expected, tmp_path
+):
+    current_test_data_dir = tests_data_dir / 'select_rename_fill_columns' / test_data_name_dir
+    yaml_file_path = current_test_data_dir / 'input' /  yaml_file
+    annotations_path = current_test_data_dir / 'input' /annotations
+    expected_path = current_test_data_dir / 'expected' / expected
+    output_path = tmp_path / 'out.parquet'
+    cli_runner = CliRunner()
+    cli_parameters = [
+        'select-rename-fill-annotations',
+        yaml_file_path.as_posix(),
+        annotations_path.as_posix(),
+        output_path.as_posix(),
+        ]
+    result = cli_runner.invoke(annotations_cli, cli_parameters, catch_exceptions=False)
+    assert result.exit_code == 0
+    written_results = pd.read_parquet(output_path)
+    expected_results = pd.read_parquet(expected_path)
+    assert written_results.shape == expected_results.shape
+    assert_frame_equal(written_results, expected_results[written_results.columns], check_exact = False)
+
+
+
+# @pytest.mark.parametrize(
```

Comment:
Not used?

Reply:
The whole test is commented out 
![image](https://github.com/PMBio/deeprvat/assets/135472/9fa73e47-1ef2-4fd7-a288-8ed0044bca8a)